sacrebleu/sacrebleu.py

```diff
@@ -634,20 +667,160 @@ def corpus_bleu(sys_stream: Union[str, Iterable[str]],
+                    paired_ref_len        += paired_closest_len
+
+                for n in range(NGRAM_ORDER):
+                    total[n]   += sent_total[n]
+                    correct[n] += sent_correct[n]
+                    if paired_significance_test:
+                        paired_total[n]   += paired_sent_total[n]
+                        paired_correct[n] += paired_sent_correct[n]
+            
+            trial_bleu = compute_bleu(correct, total, sys_len, ref_len, smooth_method=smooth_method, smooth_value=smooth_value, use_effective_order=use_effective_order)
+            trial_runs.append(float(str(trial_bleu).split('=')[1].strip().split(' ')[0])) # only the bleu score
+            if paired_significance_test:
+                paired_trial_bleu = compute_bleu(paired_correct, paired_total, paired_sys_len, paired_ref_len, smooth_method=smooth_method, smooth_value=smooth_value, use_effective_order=use_effective_order)
+                paired_trial_runs.append(float(str(paired_trial_bleu).split('=')[1].strip().split(' ')[0])) 
+                
+        z     = 1.96 
+        sqrtn = sqrt(bootstrap_trials)
+
+        if significance_value != 0.05:
+            try:
+                import scipy.stats
+                z = scipy.stats.norm.ppf(1-significance_value/2)
+
+            except Exception:
+                print("scipy stats package not available")
+                print("using 0.05 as alpha")
+            
+        xbar  = round(mean(trial_runs), 4)
+        med   = round(median(trial_runs), 4)
+        s     = stdev(trial_runs)
+        ci    = round(z * s / sqrtn, 4)
+        if paired_significance_test:
+            paired_xbar = round(mean(paired_trial_runs), 4)
+            paired_med  = round(median(paired_trial_runs), 4)
+            paired_s    = stdev(paired_trial_runs)
+            paired_ci   = round(z * paired_s / sqrtn, 4)
+
+        print("Confidence Interval of sacreBLEU for System1 with alpha {} : {} +- {} (median {})".format(significance_value, xbar, ci, med))
+        if paired_significance_test:
+            print("Confidence Interval of sacreBLEU for System2 with alpha {} : {} +- {} (median {})".format(significance_value, paired_xbar, paired_ci, paired_med))
+
+        if paired_significance_test:
+            sys1_win = 0
+            sys2_win = 0
+            tie      = 0
+            for i in range(bootstrap_trials):
+                if trial_runs[i] > paired_trial_runs[i]:
+                    sys1_win += 1
+                elif trial_runs[i] == paired_trial_runs[i]:
+                    tie +=1
+                else:
+                    sys2_win += 1    
+            sys1_win /= bootstrap_trials ; sys2_win /= bootstrap_trials
+            if sys2_win > sys1_win:
+                print("System2 is superior with p-value {}".format(round(1-sys2_win, 3)))
+            else:
+                print("System1 is superior with p-value {}".format(round(1-sys1_win, 3)))
+            
+    return orig_bleu
```

Comment:
It would be nice if the bootstrap statistics (e.g. p-value) are available through the API, i.e. returned in the BLEU object. However, this is just a suggestion - I haven't thought about it much and it could be done in a separate PR.